golang/cmd/gcp_key_gen/README.md

```diff
@@ -0,0 +1,32 @@
+# Script to Create Tink Hybrid Keys using Envelope Encryption with GCP
+
+This script creates a [Tink Hybrid](https://github.com/google/tink/blob/master/docs/PRIMITIVES.md#hybrid-encryption) ([ECIES](https://en.wikipedia.org/wiki/Integrated_Encryption_Scheme) with AEAD) key and outputs the public key in plaintext to an input file, as well as the encrypted private key to another input file. The private key is encrypted using a key hosted on
+Google Cloud (GCP). This method is commonly referred to as [Envelope Encryption](https://cloud.google.com/kms/docs/envelope-encryption).
+
+This script was inspired by the Medium post [Google Cloud KMS & Tink](https://medium.com/google-cloud/google-cloud-kms-tink-1e106156bb4e). Please read that post for more information about setting up GCP keys.
+
+## Installation:
+
+```shell
+# Clone repo
```

Comment:
A user shouldn't have to clone the repo to use this.

Reply:
True. Made READMEs more go-friendly.